jscomp/ml/parmatch.ml

```diff
@@ -426,17 +426,13 @@ let rec pretty_val ppf v =
             | _ -> true) in
       begin match filtered_lvs with
       | [] -> fprintf ppf "_"
-      | (_, lbl, _) :: q ->
-          let elision_mark ppf =
-            (* we assume that there is no label repetitions here *)
-             if Array.length lbl.lbl_all > 1 + List.length q then
-               fprintf ppf ";@ _@ "
-             else () in
+      | (_, _lbl, _) :: _q ->
+          let elision_mark _ = () in
```

Comment:
@cristianoc a bit unsure about this one, what do you think? Should we keep the old behavior of adding `_`, just convert it to ReScript syntax, or remove it like I've done here?

Reply:
Not sure what this does. Does it suggest the change that is only relevant for the warning that is not on by default?
Then sure I'd also skip it.

Reply:
Yes exactly, that's what it does.